code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm

```diff
@@ -38,9 +39,143 @@
 	..()
 	src << "<span class='notice'>You can kill and eat other drones to increase your health!</span>" //Inform the evil lil guy
 
+/*
+//Syndrones shouldn't be blocked like regular drones.
+/mob/living/simple_animal/drone/syndrone/ClickOn(atom/A, params)
+	...(A, params)
+*/
 
 /obj/item/drone_shell/syndrone
 	name = "syndrone shell"
 	desc = "A shell of a syndrone, a modified maintenance drone designed to infiltrate and annihilate."
 	icon_state = "syndrone_item"
-	drone_type = /mob/living/simple_animal/drone/syndrone
\ No newline at end of file
+	drone_type = /mob/living/simple_animal/drone/syndrone
+
+//Safedrones - Excuse the awful ClickOn proc
+/mob/living/simple_animal/drone/safedrone
+
+/mob/living/simple_animal/drone/ClickOn(atom/A, params)
+	if(world.time <= next_click)
```

Comment:
This seems snowflakey, could you not use the changeNext_move function? It essentially does this. Not sure if it would transfer to drones however.